src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -330,6 +330,7 @@ protected virtual bool IsAttributeConditionallyOmitted(NamedTypeSymbol attribute
                         }
 
                         BoundExpression boundNamedArgument = BindNamedAttributeArgument(argument, attributeType, diagnostics);
+                        this.Compilation.VariableUsePass.Visit(boundNamedArgument);
```

Comment:
> I see no tests for this code path that would demonstrate that this code makes a difference and that difference is what we would expect.

It looks like this comment is still not addressed.

---
In reply to: [116878643](https://github.com/dotnet/roslyn/pull/19444#discussion_r116878643) [](ancestors = 116878643)